### PR TITLE
ADF diagnostic configuration added

### PR DESCRIPTION
**Pull request by s-vishnjakov**

### CODE REVIEW

**Comment by s-vishnjakov** [comment]
Commit 29d7f69

**Review by s-vishnjakov — COMMENTED** [review]
It's the proper commit

File: Overview.md [hunk 1]
```diff
@@ -23,6 +23,21 @@ When the solution is just deployed, the monitoring worker does not have any acce
 Go to the deployed managed application and open the Parameters and Outputs blade. Copy the value of the monitoring Identity output.  
 < screenshot here >  
 Provide this identity with a Reader role to your subscription ([more info](https://docs.microsoft.com/en-us/azure/active-directory/managed-identities-azure-resources/howto-assign-access-portal))
+
+## Configure ADF Diagnostic Settings
+Configure diagnostic setting to send platform logs and metrics to Log Analytics Workspace.
+
+![Diagnostic Settings](Artifacts/DiagnosticSettings.PNG)
+1. Go to the menu of the created Data Factory.
```
Comment (reaver89) [inline Overview.md]: 1. Not "created" but "Data Factory whic you are going to monitor".

File: Overview.md [hunk 2]
```diff
@@ -23,6 +23,21 @@ When the solution is just deployed, the monitoring worker does not have any acce
...
+## Configure ADF Diagnostic Settings
+Configure diagnostic setting to send platform logs and metrics to Log Analytics Workspace.
+
+![Diagnostic Settings](Artifacts/DiagnosticSettings.PNG)
+1. Go to the menu of the created Data Factory.
+2. On the Data Factory's blade menu select "Diagnostic Settings". Diagnostic settings menu will opens.
+3. Click on "+Add diagnostic Setting".
+4. Specify a name for the Diagnostic setting. 
+5. Select the required categories to send: ActivityRuns, PipelineRuns, TriggerRuns, AllMetrics.
+6. Select the subscription that contains VIAcode-Azure-Data-Factory monitoring and workspace itself.
```
Comment (reaver89) [inline Overview.md]: I don't see info about  "Destination Details" , which destination table I should select and so on

File: Overview.md [hunk 3]
```diff
@@ -23,6 +23,21 @@ When the solution is just deployed, the monitoring worker does not have any acce
 Go to the deployed managed application and open the Parameters and Outputs blade. Copy the value of the monitoring Identity output.  
 < screenshot here >  
 Provide this identity with a Reader role to your subscription ([more info](https://docs.microsoft.com/en-us/azure/active-directory/managed-identities-azure-resources/howto-assign-access-portal))
+
+## Configure ADF Diagnostic Settings
```
Comment (reaver89) [inline Overview.md]: This section is like  "prerequisite", withou Diagnostic settings configuration Workbooks will not work. It's better to put it into the beginning of the guide

**Review by reaver89 — CHANGES_REQUESTED** [review]

File: Overview.md (hunk 3 above)
Comment (s-vishnjakov) [reply Overview.md]: Got it.

File: Overview.md (hunk 1 above)
Comment (s-vishnjakov) [reply Overview.md]: edited

**Review by reaver89 — APPROVED** [review]

**Pull request merged**